envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/permissions/IncomingPermissionsLoggingModeTest.kt

```diff
@@ -602,7 +619,8 @@ internal class IncomingPermissionsLoggingModeTest : EnvoyControlTestConfiguratio
             path = "/unlisted-endpoint",
             method = "GET",
             clientName = "",
-            clientIp = echoEnvoy.gatewayIp()
+            clientIp = echoEnvoy.gatewayIp(),
```

Comment:
Can you add a test which verify that `requestId` has been set?

Reply:
Added